### PR TITLE
Add a check for the setting of watch flag in config file

### LINKED ISSUE #1074: Config variable «watch» in config.yaml doesn't work
I has this in my config.yaml:

```
# watch filesystem for changes and recreate as needed
watch: true
```
According to [Configuring Hugo](http://gohugo.io/overview/configuration/) documentation.
But, it didn't seem to work. I still need to use options `-w` or `-watch` in command line to watching files to work.

After commenting this in a [a forum thread](http://discuss.gohugo.io/t/watch-configuration-variable-doesnt-work/1092), I realized that maybe (and only maybe), considering that the documentation only says that:

```
# watch filesystem for changes and recreate as needed
watch:                      false
```

it's aimed for people that deploys their site with the hugo's built-in server (instead of nginx, apache...), and it's intended to **only** being set to `false`, **never** to `true`, just to deactivate the watch in production.

If that's the case, I think that the comment in that documentation:

```
# watch filesystem for changes and recreate as needed
```

Should be changed for something clearer, because now it'ss pretty confuse. Someone (that's me!) can guess that, if you set it to `true`, it will watch for changes without the need of explicitly using `-w` or `-watch` option in commandline.

Something like:

```
# disable watch of filesystem for changes (handy for production environments)
```

would be much clearer.

Just in case it is in fact a bug (and it should work also for activate `watch`):
I'm using archlinux and `hugo version`

>Hugo Static Site Generator v0.13 BuildDate: 2015-04-20T21:17:06+02:00

(installed from AUR package, [hugo](https://aur.archlinux.org/packages/hugo/)).

### PR DESCRIPTION
**Pull request by scottcwilson**
Fixes #1074

### CODE REVIEW

**Comment by bep** [comment]
Oh, and one last thing ... You do very good work, but remember to run "go fmt" on the code before commit.

I use LiteIde for Go coding, but I know other editors too have "go fmt on save".

**Comment by scottcwilson** [comment]
Rats!  Sorry, I could have sworn I did.  I will double check the README before my next commit.  Thanks for your patience.

File: commands/server.go [hunk 1]
```diff
@@ -67,6 +67,10 @@ func server(cmd *cobra.Command, args []string) {
 		viper.Set("DisableLiveReload", disableLiveReload)
 	}
 
+	if viper.GetBool("watch") {
+    	serverWatch = true
+    }
```
Comment (bep) [inline commands/server.go]: If you can `go fmt` this and move it below the next conditional (of no practical difference, but as it is now it is slightly confusing to read).

**Comment by scottcwilson** [comment]
Closing this PR; new one opened #1286.

**Comment by bep** [comment]
Closing and open a new PR is good for the project, as it shows a lot of activity ... but:

I notice you do your changes on your own master branch.

* Please create a new branch for every new pull request (a Git Branch is cheap)
* When doing changes, just do a "commit --amend" or similar followed by "git push -f" (force push, nobody cares about the history of a temporary branch)

**Comment by scottcwilson** [comment]
Got it. Thanks.

**Pull request closed without merging**